packages/design-system/src/helpers/tokens/anchor.json

```
@@ -0,0 +1,76 @@
+{
+  "base": {
+    "color": {
+      "text": {
+        "default": {
+          "value": "{primary.500}",
+          "type": "color"
+        },
+        "hover": {
+          "value": "{primary.600}",
+          "type": "color"
+        },
+        "focus": {
+          "value": "{primary.300}",
+          "type": "color"
+        }
+      }
+    }
+  },
+  "bottomDecoration": {
```

```suggestion
  "decorationBottom": {
```
I think it doesn't work ;) In Anchor component you have `decorationBottom`.